plugins/modules/certificate_complete_chain.py

```diff
@@ -196,6 +197,11 @@ def is_parent(module, cert, potential_parent):
                 cert.cert.tbs_certificate_bytes,
                 cryptography.hazmat.primitives.asymmetric.ec.ECDSA(cert.cert.signature_hash_algorithm),
             )
+        elif isinstance(public_key, cryptography.hazmat.primitives.asymmetric.ed25519.Ed25519PublicKey):
```

Comment:
I was thinking about adding ed448 as well, since they are so similar. Wasn't sure if that'd be better as a separate PR or part of this one.

I'll add ed448 and the changelog fragment.  It might be a few days before I can test the ed448 addition.

Reply:
Doing it in the same PR is fine, after all it's a pretty small change :) Thanks for doing this!